### DISCUSSION TITLE
LINE也要發NFT了！

### DISCUSSION THREAD

**Discussion by vvvioletta** [1101-GS4719-C/Python-Community-GS4719-C#204] [General]
日本通訊軟體LINE宣布，將在2022年推出LINE NFT平台，LINE用戶可以互相傳送NFT，同時在次級市場販賣時，原創作者也能獲得分潤。
NFT的熱度持續上升，所有產業都希望在這波浪潮中取得一席之地，LINE的這波佈局除了打造自身的區塊鏈生態圈，我想不久的未來，LINE進軍元宇宙將成為下一個話題，把所有社群整合成LINE元宇宙也說不定。
[「LINE NEXT」公司成立！一次看懂LINE的NFT布局](https://www.bnext.com.tw/article/66805/line-nft)

**Comment by Tomlord1122** [comment]
感覺越來越多公司開始投資NFT，或許他們也不清楚之後的應用等等，但有越來越多大公司投資，我想會讓一般人也會更願意去了解這項技術，像是前幾個禮拜adiddas在sandbox花了五千萬買了一塊地一樣。

**Comment by 107602505** [comment]
如果line未來也發了NFT想必一定會擴大市場，畢竟line是除了台灣還有日本及韓國幾乎人人都在使用的通訊軟體。

**Comment by chia-kai-chang** [comment]
同學們真的是太強大了，這是很新的消息啊～🥳

**Comment by ghost** [comment]
NFT感覺越來越紅了欸，不知道之後學校會不會有相關的課程，有的話應該會蠻有趣的

**Comment by Jen-Siang** [comment]
常常聽到NFT，但這還是第一次認真去看NFT的文章，還蠻有趣的

**Comment by chun1203** [comment]
虛擬貨幣和NFT感覺越來越火，是個值得研究的議題

**Comment by EricAgumonJou** [comment]
我還補充一下什麼是NFT，可能有同學沒聽過。
NFT的全名是「非同質化代幣」（Non-Fungible Token），簡單來說，NFT跟你我熟悉的比特幣這類加密貨幣，概念上是相反的。

**Reply by EricAgumonJou** [reply]
每個人所持有的比特幣，全都是一模一樣、價值相同，交易時也可以被分拆，不需要購買完整一顆比特幣；NFT則相反，每一個NFT都是獨一無二、不可相互替代的，而且交易時不可以被分拆。

**Comment by Isongzhe** [comment]
我自己目前還是對NFT保持不看好的看法。尤其目前許多遊戲都說要結合NFT，個人不太能接受。

**Reply by chia-kai-chang** [reply]
我目前對NFT也是不看好，不太清楚要怎麼防止山寨?
區塊練不能山寨，但是可以用人的影音圖片素材，自創另一個NFT阿

**Comment by YuHengChen54** [comment]
感覺NFT真的是未來的趨勢阿，不管甚麼產業都要開始發展NFT了

**Comment by pekoranoinu** [comment]
NFT雖然火熱，但是隨時可能會崩盤。
